docs/pages/versions/unversioned/sdk/background-fetch.md

```diff
@@ -11,7 +11,9 @@ import TableOfContentSection from '~/components/plugins/TableOfContentSection';
 
 | Android Device | Android Emulator | iOS Device | iOS Simulator | Web |
 | -------------- | ---------------- | ---------- | ------------- | --- |
-| ✅             | ✅               | ✅         | ✅            | ✅  |
+| ✅ *           | ✅ *             | ✅ *       | ✅ *          | ✅  |
```

Comment:
```suggestion
| ✅             | ✅               | ✅         | ✅            | ✅  |
```

Reply:
with the "known issues" section added, I don't think we need the asterisks

Reply:
:+1:

I'll send the updated doc shortly.